static/beta/prod/navigation/rhel-navigation.json

```diff
@@ -1,318 +1,369 @@
             }
-        ]
-      },
-      {
-        "title": "Product Materials",
-        "expandable": true,
-        "routes": [
-          {
-            "id": "productMaterialDocumentation",
-            "title": "Documentation",
-            "isExternal": true,
-            "href": "https://access.redhat.com/documentation/en-us/red_hat_insights/"
-          },
-          {
-            "id": "securityInformation",
-            "title": "Security Information",
-            "href": "https://www.redhat.com/en/technologies/management/insights/data-application-security",
-            "isExternal": true
-          },
-          {
-            "id": "apis",
-            "appId": "apiDocs",
-            "title": "APIs",
-            "href": "/docs/api"
-          }
-        ]
-      }
-    ]
+          ]
+        },
+        {
+          "id": "ros",
+          "appId": "ros",
+          "title": "Resource Optimization",
+          "href": "/insights/ros",
+          "product": "Red Hat Insights",
+          "description": "Optimize your public cloud-based Red Hat Enterprise Linux systems based on CPU, memory, and disk input/output performance."
+        }
+      ]
+    },
+    {
+      "title": "Automation Toolkit",
+      "expandable": true,
+      "routes": [
+        {
+          "id": "remediations",
+          "appId": "remediations",
+          "title": "Remediations",
+          "href": "/insights/remediations",
+          "product": "Red Hat Insights",
+          "description": "Use Ansible Playbooks to resolve configuration, security, and compliance issues identified on your Red Hat Enterprise Linux systems."
+        },
+        {
+          "id": "tasks",
+          "appId": "tasks",
+          "title": "Tasks",
+          "href": "/insights/tasks",
+          "product": "Red Hat Insights"
+        }
+      ]
+    },
+    {
+      "id": "registerSystems",
+      "appId": "registration",
+      "title": "Register Systems",
+      "filterable": false,
+      "href": "/insights/registration",
+      "product": "Red Hat Insights",
+      "description": "Register your systems with the Red Hat Insights Client to view them on the Red Hat Hybrid Cloud Console."
+    },
+    {
+      "title": "Product Materials",
+      "expandable": true,
+      "routes": [
+        {
+          "id": "productMaterialDocumentation",
+          "title": "Documentation",
+          "isExternal": true,
+          "href": "https://access.redhat.com/documentation/en-us/red_hat_insights/"
+        },
+        {
+          "id": "securityInformation",
+          "title": "Security Information",
+          "href": "https://www.redhat.com/en/technologies/management/insights/data-application-security",
+          "isExternal": true
+        },
+        {
+          "id": "apis",
+          "appId": "apiDocs",
+          "title": "APIs",
+          "href": "/docs/api"
+        }
+      ]
+    }
```

Comment:
I'm not seeing any of these in the mocks. Perhaps I'm looking at the wrong mocks?

Reply:
Yup, this does not exist in the mockups. this is what PMs wanted